src/Resources/views/storefront/page/checkout/confirm/confirm-payment.html.twig

```diff
@@ -13,7 +13,10 @@
              data-payment-finish-url="{{ adyenFrontendData.paymentFinishUrl }}"
              data-payment-error-url="{{ adyenFrontendData.paymentErrorUrl }}"
              data-edit-payment-url="{{ adyenFrontendData.editPaymentUrl }}"
+             data-state-data-url="{{ adyenFrontendData.stateDataUrl }}"
+             data-cart-url="{{ adyenFrontendData.cartUrl }}"
              data-order-id="{{ adyenFrontendData.orderId }}"
+             data-currency="{{ adyenFrontendData.currency }}"
```

Comment:
I think you will need to pass the cart value in minor units as well from the backend